flipperdevices/flipperzero-firmware
# Embed assets in elf file

# What's new

- A way to embed large constant data inside the application.

# Verification 

- Run the example_apps_assets application, check that the data will be shown in the log.
- Launch other applications from the sd-card, check that they are launched normally.

# Checklist (For Reviewer)
